packages/backend/src/index.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Required External Modules
+ */
+
+import * as dotenv from 'dotenv';
+import express from 'express';
+import cors from 'cors';
+import helmet from 'helmet';
+
+dotenv.config();
+
+/**
+ * App Variables
+ */
+
+if (!process.env.PORT) {
+  process.exit(1);
```

Comment:
Make it default to something sensible instead :p

Reply:
It does, it's in.env

Reply:
Where's that .env tho?

Reply:
Ah dang, it should have been in here